src/commands/protocols/common/find-nodes-command.js

```diff
@@ -75,7 +73,7 @@ class FindNodesCommand extends Command {
         const closestNodes = await this.shardingTableService.findNeighbourhood(
             blockchainId,
             keyword,
-            await this.blockchainModuleManager.getR2(blockchainId),
+            await this.blockchainModuleManager.getR2(),
```

Comment:
Why we are not sending blockchain id here? There is a possibility that for different blockchains we will have different value for R1 and R2

Reply:
correct, I'll change this